tests/integration/mixer/util.go

```diff
@@ -150,6 +150,27 @@ func SendTraffic(ingress ingress.Instance, t *testing.T, msg, url string, calls
 	return res
 }
 
+func SendTrafficOrFailTillRequestedReturnStatus(ingress ingress.Instance, t *testing.T, msg, url string, calls int64,
```

Comment:
nit: the name is a little strange.  Maybe something like `SendTrafficAndWaitForExpectedStatus`?

Reply:
Thanks! Fixed :)